src/sources/kafka.rs

```diff
@@ -76,6 +76,10 @@ pub struct KafkaSourceConfig {
     /// The consumer group name to be used to consume events from Kafka.
     group_id: String,
 
+    /// The consumer group instance (member) id to use in static membership configurations.
```

Comment:
I might change these docs to refer to the default behavior from the beginning, instead of referencing the unset behavior at the end. This seems to imply that this is an important thing to change. 

Something like this: "Override dynamic membership and broker assignment behavior with static membership, using a group instance (member) id"

Reply:
sounds good. I'll tweak and push shortly.